src/AppBundle/DataFixtures/ORM/LoadMediaData.php

```diff
@@ -54,6 +62,8 @@ public function load(ObjectManager $manager)
                 $mediaObj->setRestaurant($this->getReference("restaurant-" . $media["restaurantSlug"]));
             }elseif(isset($media["userSlug"])){
                 $mediaObj->setUser($this->getReference("user-" . $media["userSlug"]));
+            }elseif(isset($media["postSlug"])){
+                $this->addReference($media["postSlug"], $mediaObj);
```

Comment:
can you change it to 
$this->addReference("user-" . $media["postSlug"], $mediaObj);
The "user-" part is to identify the slug, because we will maybe add other picture owners